public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-loader.php

```diff
@@ -397,6 +398,37 @@ public function register_rest_public_fields() {
 				)
 			);
 		}
+
+		register_rest_field(
+			'wordcamp',
+			'session_start_time',
+			array(
+				'get_callback' => function( $object, $field_name ) {
```

Comment:
It might be worth renaming this field to something like `first_session_start_time` and/or adding a `description` parameter to the field schema to better publicly document what this field is for.

Reply:
Added a description in https://github.com/WordPress/wordcamp.org/pull/410/commits/a628240e37b7933864dbf578383a60b59b4eee13 — I'd prefer to keep the field name shorter, and I think it's still descriptive enough without `first`.